kansha/board/comp.py

```diff
@@ -840,6 +850,21 @@ def search(self, query):
         else:
             self.card_matches = set()
 
+    @classmethod
```

Comment:
These method can't be classmethods because they don't return Boards instance but DataBoards instance. So, they should either be classmethods of DataBoard or static methods of Board.